app/main.py

```
@@ -14,11 +17,17 @@
+
 @app.get("/")
 async def root():
     return {"message": "Welcome to AstroAPI! Use /docs to explore the API"}
 
-
 @app.get("/hello/{name}")
 async def say_hello(name: str):
     return {"message": f"Hello {name}"}
+
+@app.get("/healthcheck")
+async def healthcheck(db: AsyncSession = Depends(get_db)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)